charts/k3s/values.yaml

```diff
@@ -150,7 +150,7 @@ syncer:
 # Virtual Cluster (k3s) configuration
 vcluster:
   # Image to use for the virtual cluster
-  image: rancher/k3s:v1.25.3-k3s1
+  image: rancher/k3s:v1.26.0-k3s1
```

Comment:
There is a slightly newer stable tag - [v1.26.0-k3s2](https://hub.docker.com/layers/rancher/k3s/v1.26.0-k3s2/images/sha256-08f13056ad6755948f0ad6658ec7e04356b52351c04559956b3b9527910ffd8b?context=explore), this But from the release notes the fixes seem irrelevant to vcluster, so just FYI :)

Reply:
Oh Yes!!   What does the 's2' in the end of the image stand-for. I saw that all our previous version images were s1, hence I took the latest 's1' image.

Reply:
That is just a suffix that k3s maintainers can increment when releasing a new build of the same Kubernetes release.

Reply:
Ah!! Got it. Will keep this in mind. Thanks @matskiv. :slightly_smiling_face:  :raised_hands: